source/_integrations/roomba.markdown

```diff
@@ -95,3 +95,9 @@ roomba:
 ### Retrieving your credentials
 
 Please refer to [here](https://github.com/NickWaterton/Roomba980-Python#how-to-get-your-usernameblid-and-password) or [here](https://github.com/koalazak/dorita980#how-to-get-your-usernameblid-and-password) to retrieve both the BLID (username) and the password.
+
+For Home Assistant running in Docker the following command uses the [roombapy](https://github.com/pschmitt/roombapy/blob/e70480128635657b400e503cf3de96302a9b6363/roomba/entry_points.py#L20) pacakge deployed with Home Assistant to retrieve the BLID (username) and password:
```

Comment:
# Fixed

* `Home Assistant running in Docker` -> `Home Assistant Container`
* `pacakge` -> `package`

# Discussion

* I added the link to the `roombapy` codebase in the original commit to make it easier for future users to understand what the command is invoking in case the command stops working. My expectation is that this command documentation is a short/medium term stopgap until someone implements running the `roombapy` password function from the Home Assistant UI.
* I've changed the link to the `roombapy` codebase so that it points to the main page. The main `roombapy` page is helpful because it says that the package is a fork of the previously mentioned `NickWaterton/Roomba980-Python` package and contains the `roombapy` specific instructions for retrieving the credentials. I was only able to figure out that the `roombapy` package is used by Home Assistant by diving into the Home Assistant code. This documentation is trying to help future user avoid needing to read the code to figure this out.
* I've moved the details of "why" about this command to a note that can more clearly be skipped by novice users.